app/messages/Messages.scala

```diff
@@ -346,20 +346,31 @@ object Messages {
       english = "Payment summary"
     )
 
+    val `Can you make an upfront payment?`: Message = Message(
```

Comment:
the question and answer should always be the same as the title and answer on the actual page - on the summary page template then you should be able to just reuse what's in the `UpfrontPayment` object in this file

Reply:
Sorry I don't understand

Reply:
Oh wait I think I do, sorry

Reply:
the same holds for the `Yes` message as well - there's one above you can use already

Reply:
updated